packages/cloud-functions/src/cico/utils.ts

```diff
@@ -1,8 +1,87 @@
-export function getFirebaseAdminCreds(admin: any) {
+import * as admin from 'firebase-admin'
+import { v4 as uuidv4 } from 'uuid'
+const { BigQuery } = require('@google-cloud/bigquery')
+
+const gcloudProject = process.env.GCLOUD_PROJECT
```

Comment:
I moved some of this BigQuery initialization logic to a `bigQuery.ts` file in this PR https://github.com/celo-org/wallet/pull/164 which is about to get merged. Might get a conflict with that one.